src/mjolnir/elevationbuilder.cc

```diff
@@ -315,11 +315,16 @@ namespace mjolnir {
 
 void ElevationBuilder::Build(const boost::property_tree::ptree& pt,
                              std::deque<baldr::GraphId> tile_ids) {
-  auto elevation = pt.get_optional<std::string>("additional_data.elevation");
+  boost::optional<std::string> elevation = pt.get_optional<std::string>("additional_data.elevation");
+  boost::optional<bool> addElevation = pt.get_optional<bool>("additional_data.add_elevation_in_tiles");
```

Comment:
we need to add this to `valhalla_build_config`. also we should just stick it in the top level `mjolnir` config and call it something more specific like `store_elevation_along_edges`